nservicebus/rabbitmq/routing-topology_support-delayed-delivery_rabbit_[4,].partial.md

```diff
@@ -0,0 +1,7 @@
+### Delayed delivery
+
+Starting with Version 4.3, it is possible to support [delayed delivery](delayed-delivery.md) without requiring the [timeout manager](/nservicebus/messaging/timeout-manager.md) by implementing `ISupportDelayedDelivery` in addition to `IRoutingTopology` in the routing topology class.
```

Comment:
👍 for changing the version wording, but the other change isn't correct. In this overall section of the page, we aren't talking about the RabbitMQ transport, we are talking specifically about an end-user creating a custom routing topology by implementing `IRoutingTopology`. This new section is about augmenting this implementation by also implementing `ISupportDelayedDelivery`.

Based on that, yes we do need to go into this level of detail here, because it is information that someone implementing the interface needs.